library/modules/bigip_device_auth.py

```diff
@@ -52,7 +52,6 @@
       port:
         description:
           - The port of the server.
-        default: 49163
```

Comment:
i know i said no defaults but we just don't remove stuff without deprecation

Reply:
So what's the rule of thumb on how to deprecate a default?

Reply:
you would announce deprecations with this method:
https://github.com/F5Networks/f5-ansible/blob/devel/library/modules/bigip_gtm_wide_ip.py#L743

There is no set recipe as each module does it differently but the idea is for the Ansible to announce deprecation at the end of module run. I keep the code relatively tech debit free so i removed some good examples already on how to do this.